comp/swerve100/README.md

```diff
@@ -2,4 +2,70 @@
 
 This is the main comp bot repo.
 
-Part of the code here was (somewhat distantly) derived from 254 code, and other parts were inspired by Roadrunner.
\ No newline at end of file
+Part of the code here was (somewhat distantly) derived from 254 code, and other parts were inspired by Roadrunner.
+
+# Getting Started
+
+## Preparing Your Dev Env
+TODO.
+
+The rest of the instructions assume that you have the following setup:
+1. Git `Team100/all24` repo checked out.
+1. Custom WPI VSCode with extensions installed.
+1. `swerve100` workspace opened.
+
+## Building the RoboRIO Code
+Use the WPI extensions to run the `gradlew` build:
+1. Press `CTRL + SHIFT + P` to get the VSCode Command Mode.
+1. Search for `Build Robot Code`
+1. Select `swerve100` when it asks.
+1. A terminal should pop up that runs the build. If successful, you will see this:
+    ```
+    BUILD SUCCESSFUL in 2s``
+    6 actionable tasks: 1 executed, 5 up-to-date
+    Watched directory hierarchies: [C:\Users\Engineering Student\src\all24\comp\swerve100]
+    ```
+TODO: what if the build is not successful?
+
+## Simulating the RoboRIO
+Use the WPI extensions to kick off the Simulator. Good overall instructions are available [here](https://docs.wpilib.org/en/stable/docs/software/wpilib-tools/robot-simulation/simulation-gui.html). 
+1. Press `CTRL + SHIFT + P` to get the VSCode Command Mode.
+1. Search for `Simulate Robot Code`
+1. Select `swerve100` when it asks.
+1. It will ask you what simulations to run. Make sure `Sim GUI` is selected, and nothing else.
+1. You should get the robot simulation GUI, which looks like this:
+![Simulation GUI](readme_img/sim_gui.png)
+1. Make sure you can debug your code in simulation.
+    1. If you aren't familiar with debugging in VSCode, read this: TODO.
```

Comment:
we actually don't use the debugger very much, i think mostly because you can't run it on the real robot (as far as i know), and even if you could, all the behavior is so timing-dependent that it wouldn't be useful.  instead we rely on logging and unit tests.

Reply:
OK interesting. I guess I'm not used to coding w/o access to the debugger :)